Resources/Prototypes/DeltaV/Entities/Mobs/Player/human.yml

```diff
@@ -0,0 +1,19 @@
+- type: entity # Delta-V : Part of a mid-round PirateRadioSpawn
+  noSpawn: true
+  parent: MobHumanSyndicateAgent
+  id: MobHumanSyndicateListener
+  name: Syndicate Listener
+  components:
+    - type: Loadout
+      prototypes: [SyndicateListenerGear]
+    - type: NpcFactionMember
+      factions:
+      - Syndicate
+    - type: AutoTraitor
+      giveUplink: false
+      giveObjectives: false
+#    - type: AutoImplant # If Salvage begins causing trouble with the voice mask; it's possible to "on-round-start" implant them.
+#      implants:
+#      - DeathAcidifierImplant
+    - type: PsionicBonusChance #Giving them the same chance as a Nukie at being psychic
+      multiplier: 7
```

Comment:
This allows them to directly affect the station, and should be removed

Reply:
If allowing them to hear telepathic messages is intended here, just port the old PonderingOrb (which we really should have anyways since its mentioned in some random pools), and give them that instead.
https://github.com/DeltaV-Station/Delta-v-Legacy/blob/afdc1bc74090149a5de0a1ad34432ebdcc072e4d/Resources/Prototypes/Nyanotrasen/Entities/Objects/Specific/Research/misc.yml#L1-L9